src/todo/todo.controller.ts

```diff
@@ -0,0 +1,39 @@
+import { Controller, Get, Post, Put, Delete, Patch, Param, Body, ParseIntPipe } from '@nestjs/common';
+import { TodoDto } from './dto/todo.dto';
+import { Todo } from './todo.entity';
+import { TodoService } from './todo.service';
+
+@Controller('todo')
+export class TodoController {
+  constructor(private readonly todoService: TodoService) {}
+
+  @Get()
+  getAllTodos(): Promise<Todo[]> {
```

Comment:
It would be better to use `async` `await` for these functions

Reply:
Async await has now been adopted instead of Promises